charts/sysdig/README.md

```diff
@@ -375,12 +375,14 @@ configured in the UI, and the runner automatically runs these benchmarks on the
 Note: if `nodeAnalyzer.benchmarkRunner.includeSensitivePermissions` is set to `false`, the service account will not have
 the full set of permissions needed to execute `oc` commands, which most checks in `CIS Benchmark for OS4` require.
 
-### KSPM Analyzer (Preview)
+### KSPM Analyzer & KSPM Collector (Preview)
```

Comment:
This section is under "Node Analyzer" so we shouldn't put the KSPM collector here since it's not a component of the Node Analyzer.